tests/RetryAnnotationTraitTest.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace PHPUnitRetry\Test;
+namespace PHPUnitRetry\Tests;
```

Comment:
Moved the test classes under `tests` which is by far the convention in the PHP ecosystem for the test directories. IIRC Jordi Bogianno published an article with some stats about it

Reply:
Fine by me, but I would love to see the article! Some brief googling returned no results.

Reply:
Hm can't find Jordi's article either... But found https://github.com/php-pds/skeleton_research which was one of the elements Jordi was answering to IIRC